resources/i18n/en.json

```diff
@@ -13,7 +13,7 @@
             "Configurations": "Configurations",
             "Maintenance": "Maintenance",
             "Information": "Information",
-            "About": "About",
+            "AboutBackend.AI": "About Backend.AI",
```

Comment:
In translation part, dot (.) is used to divide namespaces. How about removing dot from AboutBackend.AI? (to AboutBackendAI)